src/server/game/Battlegrounds/BattlegroundQueue.cpp

```diff
@@ -963,6 +955,31 @@ void BattlegroundQueue::BattlegroundQueueUpdate(BattlegroundBracketId bracket_id
     }
 }
 
+uint32 BattlegroundQueue::GetPlayesCountInGroupsQueue(BattlegroundBracketId bracketId, BattlegroundQueueGroupTypes bgqueue)
```

Comment:
GetPlayerCountInGroupsQueue?

Reply:
Yes

Reply:
Had to read this twice... There's a typo in there:
GetPlaye**s**CountInGroupsQueue
Should be renamed to
GetPlaye**r**CountInGroupsQueue